annlite/container.py

```diff
@@ -83,7 +83,7 @@ def __init__(
             CellTable(f'table_{c}', columns=columns) for c in range(n_cells)
         ]
 
-        self._meta_table = MetaTable('metas', data_path=data_path, in_memory=False)
+        self._meta_table = MetaTable('metas', data_path=data_path, in_memory=True)
```

Comment:
why put meta table into memory?

Reply:
This is the source of failure of the delete-then-update issue.